strawberry/printer/printer.py

```diff
@@ -561,9 +561,9 @@ def print_schema_definition(
 def print_directive(
     directive: GraphQLDirective, *, schema: BaseSchema
 ) -> Optional[str]:
-    strawberry_directive = directive.extensions["strawberry-definition"]
+    strawberry_directive = directive.extensions.get("strawberry-definition")
```

Comment:
this happens because now we have new built-in but not really built-in directives from GraphQL core